services/gitlab.service

```diff
@@ -9,6 +9,7 @@ TimeoutStartSec=0
 TimeoutStopSec=15
 Restart=always
 RestartSec=5s
+ExecStartPre=/opt/bin/gitlab-network start
```

Comment:
This will return prior to acquiring an ip address. Does that make it a race condition?